src/pages/workspace/workflows/WorkspaceWorkflowsPage.tsx

```diff
@@ -261,7 +261,7 @@ function WorkspaceWorkflowsPage({policy, betas, route, reimbursementAccount, ses
 
     const isPaidGroupPolicy = PolicyUtils.isPaidGroupPolicy(policy);
     const isPolicyAdmin = PolicyUtils.isPolicyAdmin(policy);
-    const isLoading = reimbursementAccount?.isLoading ?? true;
+    const isLoading = reimbursementAccount?.isLoading && reimbursementAccount?.achData === undefined;
```

Comment:
It seems like `achData` can be [undefined](https://github.com/Expensify/App/blob/8dd68c3292416e8a0bcfbfbf28f2a81c1d9ad0ad/src/types/onyx/ReimbursementAccount.ts#L48). Are we sure this won't result in an infinite loading spinner?

Reply:
Oh nvm my comment above, I'm not sure what I was thinking when I said that 😅 Do we need to check for `isOffline = false`?